.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: java
+jdk:
+  - openjdk7
+  - oraclejdk7
+  - oraclejdk8
+before_install:
+ - chmod +x gradlew
```

Comment:
git understand file permissions, you can just make a commit with this change and remove this line, will benefit all users and not only travis :)